src/features/errors/app-error-boundary.tsx

```diff
@@ -1,49 +1,53 @@
-import React from 'react';
-import { PopupContainer } from '@components/popup/container';
-import { Header } from '@components/header';
+import React, { useEffect } from 'react';
 import { Box, Button, CodeBlock, color, Stack } from '@stacks/ui';
-import { Prism } from '@common/clarity-prism';
 
-import { useErrorStackTraceState } from '@store/ui/ui.hooks';
+import { Header } from '@components/header';
+import { Prism } from '@common/clarity-prism';
 import { Title } from '@components/typography';
 import { ErrorBoundary, FallbackProps, useErrorHandler } from '@features/errors/error-boundary';
 import { openGithubIssue } from '@features/errors/utils';
+import { useErrorStackTraceState, useRouteHeaderState } from '@store/ui/ui.hooks';
 
 function ErrorFallback({ error, resetErrorBoundary }: FallbackProps) {
   const [value] = useErrorStackTraceState();
+  const [_, setRouteHeader] = useRouteHeaderState();
+
+  useEffect(() => {
```

Comment:
Maybe we could add a custom hook for setting those Headers

Reply:
Yeah, that is a good idea, thx!